tfx/components/infra_validator/model_server_runners/kubernetes_runner.py

```diff
@@ -225,6 +250,7 @@ def _BuildPodManifest(self) -> k8s_client.V1Pod:
     result = k8s_client.V1Pod(
         metadata=k8s_client.V1ObjectMeta(
             generate_name=_MODEL_SERVER_POD_NAME_PREFIX,
+            annotations=self._serving_pod_manifest_overrides.annotations,
```

Comment:
Regarding your error: maybe we should wrap this value inside `dict(self._serving_pod_manifest_overrides.annotations)` to make it serializable by k8s REST client.

Reply:
That fixed it, great catch!